examples/param_parsers.py

```diff
@@ -0,0 +1,188 @@
+# BSD 3-Clause License
+#
+# Copyright (C) 2024, LAAS-CNRS.
+# Copyright note valid unless otherwise stated in individual files.
+# All rights reserved.
+
+import hppfcl
+import numpy as np
+import pinocchio as pin
+import yaml
+
+
+class ParamParser:
+    def __init__(self, path: str, scene: int):
```

Comment:
```suggestion
    def __init__(self, path: pathlib.Path, scene: int):
```
Path should always be handled with `pathlib` and not `str`